### PR TITLE
ZCS-13390 : zmservicectl utility for license daemon service

### PR DESCRIPTION
**Pull request by umagmrit**
zmservicectl utility to control license services.
```
$ zmservicectl service_name {star|stop|restart|status}
$ zmservicectl zmlicenseservice start|stop|restart|status
```

### CODE REVIEW

File: src/bin/zmservicectl [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	echo -n "Starting license daemon service..."
+	if [ $licenseDaemonServiceRunning = 1 ]; then
+		echo "license daemon service is already running."
+		return
+	fi
+	if [ "x${licenseDaemonServicepid}" != "x" ]; then
+		kill -9 ${licenseDaemonServicepid}
+	fi
+	if [ -f ${LD_LIBRARY_PATH}/libShafer-dev-linux64.so ]; then
+		export LD_LIBRARY_PATH; nohup ${ZMJAVA} -jar $LICENSE_DAEMON_SERVICE --spring.profiles.active=dev --server.port=$LICENSE_DAEMON_SERVICE_PORT > $LICENSE_DAEMON_SERVICE_OUTPUT 2>&1 &
```
Comment (RakeshAMore) [inline src/bin/zmservicectl:53]: Let's not pass the profile here. we can set the env variable for this

Comment (RakeshAMore) [reply src/bin/zmservicectl:53]: ok. my bad.

**Review by RakeshAMore — APPROVED** [review]

**Review by dasiyogesh — APPROVED** [review]

File: src/bin/zmservicectl [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+startLicenseDaemonService()
+{
+	err=0
+	checkLicenseDaemonServiceRunning
+	echo -n "Starting license daemon service..."
+	if [ $licenseDaemonServiceRunning = 1 ]; then
+		echo "license daemon service is already running."
+		return
+	fi
+	if [ "x${licenseDaemonServicepid}" != "x" ]; then
```
Comment (silentsakky) [inline src/bin/zmservicectl:49]: if service is already running then it will not come to this point, so when exactly we need to kill process?

Comment (umagmrit) [reply src/bin/zmservicectl:49]: right, this is not required

**Review by silentsakky — APPROVED** [review]

**Pull request merged**